config/desktop/bullseye/environments/xmonad/config_base/packages.uninstall

```diff
@@ -0,0 +1 @@
+../../../../focal/environments/xmonad/config_base/packages.uninstall
```

Comment:
I would prefer to have link logic this way - if its Ubuntu based, we link to main Ubuntu, Focal in our case, for Debian, to Buster. For most of those linking its the same. Not a deal breaker.

Reply:
I have never used links and I forgot to rn the debian to  ubuntu links . but Iwill have to learn how you do your linking